test/unit/widgets/dragAndDrop.test.ts

```diff
@@ -1,7 +1,11 @@
 import * as RdfLib from 'rdflib'
 import { JSDOM } from 'jsdom'
 
-import { makeDropTarget, makeDraggable, uploadFiles } from '../../../src/widgets/dragAndDrop'
```

Comment:
we could remove this file, although it is just a test file that I will change soon as well.